Documentation/install/system_requirements.rst

```diff
@@ -151,22 +151,24 @@ by adding the following to the helm configuration command line:
 
 .. _features_kernel_matrix:
 
-Advanced Features and Required Kernel Version
-=============================================
-Cilium requires Linux kernel 4.9.17 or higher, however development on additional
-kernel features and functionality continues to progress in the Linux community.
-Some Cilium features and functionality are dependent on newer kernel versions.
-These additional Cilium features and functionality are enabled by upgrading to
-a later kernel version as detailed below:
-
-======================= ===============================
-Cilium Feature          Minimum Kernel Version
-======================= ===============================
-IPv4 fragment tracking  >= 4.10
-:ref:`cidr_limitations` >= 4.11
-:ref:`host-services`    >= 4.19.57, >= 5.1.16,  >= 5.2
-:ref:`kubeproxy-free`   >= 4.19.57, >= 5.1.16,  >= 5.2
-======================= ===============================
+Required Kernel Versions for Advanced Features
+==============================================
+
+Cilium requires Linux kernel 4.9.17 or higher; however, development on
+additional kernel features continues to progress in the Linux community. Some
+of Cilium's features are dependent on newer kernel versions and are thus
+enabled by upgrading to more recent kernel versions as detailed below.
+
+============================= ===============================
+Cilium Feature                Minimum Kernel Version
+============================= ===============================
+:ref:`concepts_fragmentation` >= 4.10
+:ref:`cidr_limitations`       >= 4.11
+:ref:`host-services`          >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`kubeproxy-free`         >= 4.19.57, >= 5.1.16,  >= 5.2
+:ref:`session-affinity`       >= 5.7
+BPF-based proxy redirection   >= 5.7
```

Comment:
That line should also be removed. It's a v1.9 feature.

Reply:
Thanks for catching this! Pushed a new version.